src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -93,6 +109,8 @@ export const ReviewRoute: FC<ReviewProps> = props => {
           .handleCardAction(orderOrError.actionData.clientSecret)
           .then(result => {
             if (result.error) {
+              trackErrorMessageEvent("An error occurred", result.error.message)
```

Comment:
[`handleCardAction`](https://stripe.com/docs/js/payment_intents/handle_card_action) is a Stripe API call, which is expected to return an error code in addition to an error message, so I think we can populate that here too.

(The same goes for the [`confirmCardSetup`](https://stripe.com/docs/js/setup_intents/confirm_card_setup) call below.)

Reply:
Good catch will add that thanks!